src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Adjustment.xml

```diff
@@ -39,12 +39,12 @@
         <subresourceOperations>
             <subresourceOperation name="api_orders_adjustments_get_subresource">
                 <attribute name="normalization_context">
-                    <attribute name="groups">shop:cart:read</attribute>
+                    <attribute name="groups">shop:order:read:item</attribute>
```

Comment:
`cart` and `order` separation are good one and most likely should be kept as is.